network/network.go

```diff
@@ -1,2 +1,37 @@
 // Package network is for creating internetworks
 package network
+
+import (
+	"time"
+
+	"github.com/micro/go-micro/client"
+	"github.com/micro/go-micro/server"
+)
+
+var (
+	// DefaultName is default network name
+	DefaultName = "go.micro.network"
+	// DefaultAddress is default network address
+	DefaultAddress = ":0"
+	// ResolveTime ddefines the time we periodically resolve network nodes
+	ResolveTime = 1 * time.Minute
+)
+
+// Network is micro network
+type Network interface {
+	// Name of the network
+	Name() string
+	// Connect starts the resolver and tunnel server
```

Comment:
Add `Address() string` method to the network interface to expose the `tunnel.Address()`